cmd/livepeer/livepeer.go

```diff
@@ -640,11 +641,17 @@ func main() {
 			if err != nil {
 				glog.Fatal("Error setting orch webhook URL ", err)
 			}
-			n.OrchestratorPool = discovery.NewWebhookPool(n, whurl)
+			n.OrchestratorPool = discovery.NewWebhookPool(whurl)
 		} else if len(orchURLs) > 0 {
-			n.OrchestratorPool = discovery.NewOrchestratorPool(n, orchURLs)
+			n.OrchestratorPool = discovery.NewOrchestratorPool(orchURLs)
 		} else if *network != "offchain" {
-			n.OrchestratorPool = discovery.NewDBOrchestratorPoolCache(n)
+			dbOrchPoolCache := discovery.NewDBOrchestratorPoolCache(dbh, n.Eth, roundsWatcher, n.Sender)
+			bcast := core.NewBroadcaster(n)
+			dbOrchPoolCache.PollOrchestratorInfo(bcast)
+			if originalLastSeenBlock == nil || originalLastSeenBlock.Cmp(currentRoundStartBlock) < 0 {
+				err = dbOrchPoolCache.CacheTranscoderPool(bcast)
```

Comment:
Need to handle error here. I think we should just return/exit if we hit an error here since the broadcaster probably shouldn't continue operating in on-chain mode if it is unable to compute the active set. What do you think?

Reply:
fixed in 9458ef19